pkg/controller/workload/job/job_controller.go

```diff
@@ -40,21 +38,17 @@ import (
 	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
 	"sigs.k8s.io/kueue/pkg/constants"
 	"sigs.k8s.io/kueue/pkg/controller/workload/jobframework"
-	"sigs.k8s.io/kueue/pkg/workload"
 )
 
 var (
-	ownerKey          = ".metadata.ownerReferences[batch.Job]"
 	parentWorkloadKey = ".metadata.parentWorkload"
+	gvk               = metav1.GroupVersionKind{Group: batchv1.SchemeGroupVersion.Group, Version: batchv1.SchemeGroupVersion.Version, Kind: "Job"}
```

Comment:
```suggestion
	gvk               = batchv1.SchemeGroupVersion.WithKind("Job")
```

Reply:
Done, for that I had to change the package for GVK from metav1 to schema.